src/course-search/styles/_SearchField.scss

```diff
@@ -0,0 +1,14 @@
+.fe__searchfield {
+  border-radius: 0 !important;
+}
+
+.fe__searchfield-input-box {
+  @extend .mb-2;
+  @extend .text-brand-primary !optional;
+  @extend .font-weight-normal;
+}
+
+.fe__searchfield--inverse {
+  border: 0 !important;
```

Comment:
What's the important flag overriding?

Reply:
Bootstrap almost always throws `!important` on their css classes, which annoys me greatly, because then it's hard to get enough specificity to override them. But, that's what this is doing, overriding a bootstrap class.